### DISCUSSION TITLE
Broken link: /guide/components/events.html

### DISCUSSION THREAD

**Discussion by LetMeDream** [vuejs/docs#2109] [General]
So i'm trying to read the docs about how to emit an event from a child to a parent component and a hit a broken link.

The link asks me to please let you guys know about it, but I really am at a loss as to how. I hope maybe this is the way to do it.

![download](https://user-images.githubusercontent.com/17774989/204063640-46374331-9755-4cfa-a4dd-e3abaa3b6894.png)

**Comment by skirtles-code** [comment]
Could you confirm the full URL of the page you were on? I just tried <https://vuejs.org/guide/components/events.html> but it seems to be loading correctly for me.

Perhaps it has something to do with the link you clicked. Where was that?

Thanks.

**Reply by LetMeDream** [reply]
I had actually tried both; 
1) clicking in the 'events' section directly on the sidebar nav 
![sidebar](https://user-images.githubusercontent.com/17774989/204065398-811a8eb5-68e9-452b-8e7f-9f94c33fef01.jpg)

and 2) Clicking the 'Next' button, in the previous section (Props' section)
![next](https://user-images.githubusercontent.com/17774989/204065431-a4af915d-8aaa-4f13-b96a-2ebb574f8733.jpg)

But, as you say, they are both working right now. I don't know what happened there, could have just been my browser. 
Thanks for the quick answer and I apologize for the confusion.
